Groupon UP/Groupon UP/UPViewController.swift

```diff
@@ -7,29 +7,211 @@
 //
 
 import UIKit
+import Parse
 
 class UPViewController: BaseViewController {
+    var up: UpInvitation!
+    var rsvpUsers: [PFUser?] = []
 
-    override func viewDidLoad() {
-        super.viewDidLoad()
+    private var _upContentView: UIView!
+    private var _rsvpTableView: UITableView!
 
-        // Do any additional setup after loading the view.
+    private var _messageLabel: UILabel!
+    private var _message: UITextView!
+    private var _grouponUPDateLabel: UILabel!
+    private var _grouponUPDate: UITextField!
+    private var _rsvpTableViewLabel: UILabel!
+
+    override func viewWillAppear(animated: Bool) {
+        super.viewWillAppear(animated)
+        initData()
+    }
+
+    override func initializeUI() {
+        title = "Groupon UP Detail"
+    }
+
+    override func addSubviews() {
+        view.addSubview(upContentView)
+
+        upContentView.addSubview(messageLabel)
+        upContentView.addSubview(message)
```

Comment:
@roboxue hmm, I added the `message` `UITextView`, but I am unable to edit the text, am I missing anything?